.github/workflows/ci.yml

```diff
@@ -118,6 +125,7 @@ jobs:
           # We use raw because our release process is not done on tag push (yet)
```

Comment:
I think that keeping the `latest` flavor under the condition that the workflow is running for the `master` branch means that we will keep tagging the snapshot versions as `latest` too.
I would suggest disabling the `latest` flavor altogether (`latest=false`), and instead explicitly setting the extra tag to `latest` or `snapshot` based on the proper conditions.

Reply:
my idea is to implement `latest` tag which will represent the latest stable version
that practice is proven and I would say expected... it's implemented with this condition: `type=raw,value=${{ steps.version.outputs.latestTag }},enable=${{ !endsWith(steps.info.outputs.version, '-SNAPSHOT') }}`
for your second sentence, i think we will have same result but with no consistency between versions :/
I'm open to discuss more about this 🙏 💡